src/meta/src/rpc/election_client.rs

```diff
@@ -182,14 +183,32 @@ impl ElectionClient for EtcdElectionClient {
                                 timeout.reset();
                             },
                             Ok(None) => {
-                                tracing::debug!("lease keeper for lease {} response stream closed unexpected", lease_id);
+                                tracing::warn!("lease keeper for lease {} response stream closed unexpected", lease_id);
 
                                 // try to re-create lease keeper, with timeout as ttl / 2
-                                if let Ok(Ok((keeper_, resp_stream_))) = time::timeout(Duration::from_secs_f64(ttl as f64 / 2.0), lease_client.keep_alive(lease_id)).await {
-                                    keeper = keeper_;
-                                    resp_stream = resp_stream_;
+                                // TODO: should we introduce retry with backoff here?
```

Comment:
cc @yezizp2012 @shanicky what do you think? I feel that retry may help here when there is temporary network hiccup.

Reply:
The ticker is set to 1 second, it means the keep alive stream will be created every 1 second if there's any network hiccup. Few days ago I suggest @shanicky to give a larger configuration for max lease keep interval, it would be helpful and more stable in k8s env.

Reply:
Currently, we keep trying to rebuild the lease keeper within the TTL/2 timeout after the last successful keep-alive.Regarding the issues in the longevity test, it seems that the timeout for a single keep-alive has exceeded our TTL, so retrying may not be useful.

Reply:
I see. I miss the fact the `resp_stream.message()` will keep returning `Ok(None)` if the stream is closed.